MorphicServer/Endpoint.cs

```diff
@@ -371,6 +382,11 @@ public static Database GetDatabase(this HttpContext context)
             return context.RequestServices.GetRequiredService<Database>();
         }
 
+        public static MorphicSettings GetMorphicSettings(this HttpContext context)
```

Comment:
If we go ahead with the dependency injection stuff, `MorphicSettings` could be injected to whichever endpoints need it.